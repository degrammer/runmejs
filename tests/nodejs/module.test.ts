import { test, expect } from 'vitest'

import { parse } from '../../src/index.js'

test('can import and use the package', async () => {
  expect(await parse('# Hello World\n```sh\necho "Hello World"\n```'))
    .toMatchSnapshot()
})
