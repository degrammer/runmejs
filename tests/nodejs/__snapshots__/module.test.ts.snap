// Vitest Snapshot v1

exports[`can import and use the package 1`] = `
[
  {
    "kind": 1,
    "languageId": "",
    "value": "# Hello World",
  },
  {
    "kind": 2,
    "languageId": "sh",
    "metadata": {
      "runme.dev/name": "echo-hello",
    },
    "value": "echo \\"Hello World\\"",
  },
]
`;
